widgetssdk/src/main/java/com/glia/widgets/core/dialog/DialogController.java

```diff
@@ -49,10 +49,20 @@ public boolean isOverlayDialogShown() {
         return dialogsState instanceof DialogsState.OverlayPermissionsDialog;
     }
 
+    public boolean isExitQueueDialogShown() {
+        return dialogsState instanceof DialogsState.ExitQueueDialog;
+    }
+
+    public void dismissCurrentDialog() {
+        Logger.d(TAG, "Dismiss current dialog");
+        emitDialogState(new DialogsState.NoDialog());
+        runPostponedOperation();
+    }
+
     public void dismissDialogs() {
         Logger.d(TAG, "Dismiss dialogs");
         emitDialogState(new DialogsState.NoDialog());
-        runPostponedOperation();
+        postponedOperation = null;
```

Comment:
Not sure about this change. It looks like without a call to `runPostponedOperation()` this function doesn't dismiss the dialog anymore. Is it okay?

Reply:
@andrews-moc actually, I renamed the old method to `dismissCurrentDialog()` which at the end runs the postponed one, and also created a new method `dismissDialogs()` which dismisses all dialogs including the postponed one, I'm using the second one only when user clicks `yes` on `exitFromQueue`  dialog.

Reply:
Okay ✔️